test/functional/feature_block.py

```diff
@@ -736,7 +736,7 @@ def out_value(idx):
 
         self.log.info("Reject a block with timestamp before MedianTimePast")
         b54 = self.next_block(54, self.get_staking_coin(), spend=out[15])
-        b54.nTime = b35.nTime - 1
+        b54.nTime = b33.nTime - 1
```

Comment:
Can you explain this change? I am not very familiar with feature_block.py unfortunately...

Reply:
That was necessary with a previous version of the PR. Now is not needed anymore.